src/zeek.bif

```diff
@@ -2879,10 +2879,11 @@ function to_port%(s: string%): port
     int port = 0;
     if ( s->Len() < 10 )
```

Comment:
Would it be easier to just change this to `s->Len() > 0 && s->Len() < 10`, or is there concern that the string object might not contain pure string data?

Reply:
I've pushed that simplification. Thanks.